files/build/scripts/curl

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+
+PARENT_PATH=$(dirname $0)
+REAL_COMMAND=/usr/bin/curl
+
+REAL_COMMAND=$REAL_COMMAND ${PARENT_PATH}/wget $@
```

Comment:
>$@ [](start = 47, length = 2)

It's very common to quote the argument of `curl`. Does it working?

Reply:
Fixed, thanks.